polyglot/piranha/src/utilities/tree_sitter_utilities.rs

```diff
@@ -314,6 +314,11 @@ fn _get_tree_sitter_edit(
   }
 }
 
+/// As input, `substitute_tags` requires the (i) `input_string`, (ii) substitutions for the tags (i.e. @<var>) 
+/// and whether the input string is a tree-sitter query (else it is considered as a replacement pattern).
+/// `substitute_tags` replaces the all the occurrences of a specific tag  with the corresponding string values 
+/// specified in `substitutions`, 
+/// Not it escapes newline characters for tree-sitter-queries. 
```

Comment:
`Not` -> `Note`, I assume. Also, isn't there a standard way to document arguments in Rust doc comments? https://doc.rust-lang.org/rust-by-example/meta/doc.html (the example there seems to suggest there is)

Reply:
Updated the comment to follow the documentation guidelines.